rebasehelper/specfile.py

```diff
@@ -30,7 +30,8 @@
 import shlex
 from datetime import date
 from operator import itemgetter
-from six.moves.urllib.parse import urlparse
+# TODO: remove the pylint comment once pylint with https://github.com/PyCQA/pylint/issues/551 is released
+from six.moves.urllib.parse import urlparse  # pylint: disable=import-error
```

Comment:
@thozza: No, it’s fine. But for the sake of consistency we should probably change it also in `utils.py` (import `urlopen` and `urlsplit` directly).